oms-all-deploy/nestedtemplates/omsWorkspace.json

```diff
@@ -179,7 +179,7 @@
           },
           "properties": {
             "ETag": "*",
-            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
+            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
```

Comment:
This displays as having a non-displayable character after "36". What character is it?